### ISSUE TITLE
Incorrect parameter passed to TaskHostAcquireFailed resource string

### ISSUE THREAD

**Issue by jzabroski** [microsoft/msbuild#5131]
https://github.com/microsoft/msbuild/blob/81d9a23b505474a1e7fff3fad86fa5ba2c475523/src/Build/Instance/TaskFactories/TaskHostTask.cs#L568

In .NET Core, msbuild appears to be an msbuild.dll, not an msbuild.exe.

See discussion here: https://github.com/microsoft/msbuild/issues/4332#issuecomment-586749375

**Comment by rainersigwald** [comment]
This is true but the larger issue is that .NET Framework TaskHosts aren't supported on .NET Core (and .NET Core task hosts aren't supported on .NET Framework--#4834).

If there's an way to say that in the error we should consider it rather than failing with this generic error.

**Comment by jzabroski** [comment]
Why can't .NET Core MSBuild tunnel to an MSBuild running .NET Framework?  I'm sure there are some complexities there, but is it really a bad idea?  There is already MsbuildVersionLoader - why can't there be MsBuildProgramLauncher. The main reason I see to not do this is that .NET Framework will eventually go away, so it's literally adding complexity for something nobody really wants to hang around.

**Comment by rainersigwald** [comment]
@jzabroski Yeah, it'd be possible to do using the existing TaskHost infrastructure. As with the inverse question #4834, it's probably more work in setting up and delivering all the right bits to the right places than in IPC since our IPC should already be compatible. But there hasn't yet been a compelling reason to do it relative to other work.
